modules/apps/portal-odata/portal-odata-impl/src/main/java/com/liferay/portal/odata/internal/filter/ExpressionVisitorImpl.java

```diff
@@ -227,8 +227,17 @@ public Object visitMethodExpression(
 			return _contains(
 				(EntityField)expressions.get(0), expressions.get(1), _locale);
 		}
+		else if (type == MethodExpression.Type.NOW) {
+			if (!expressions.isEmpty()) {
+				throw new UnsupportedOperationException(
+					StringBundler.concat(
+						"Unsupported method visitMethodExpression with method",
+						"type ", type, " and ", expressions.size(), "params"));
+			}
 
-		if (type == MethodExpression.Type.STARTS_WITH) {
+			return _normalizeDateLiteral(ISO8601Utils.format(_now));
```

Comment:
Hey @cgoncas ,

I was wondering if maybe we should use the user TimeZone here or the company timezone? What do you think?

Reply:
Right now all dates are format using the local timezone, and I don't feel like I want to change that behaviour right now, maybe we can open a bug to analize all the implications.

Reply:
Thanks for the clarification. Yes, this is not the moment to that change.